lib/compilers/kotlin.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2019, Compiler Explorer Authors
```

Comment:
Outdated copyright year
```suggestion
// Copyright (c) 2021, Compiler Explorer Authors
```